tests/unittests/algo/pbt/test_pbt.py

```diff
@@ -549,6 +558,58 @@ def test_suggest_num_population_size_fork_completed(self, space: Space, mocker):
         pbt_sample_mock.assert_called_with(2)
         pbt_fork_mock.assert_called_with(2)
 
+    def test_suggest_copy_working_dir(self, space, mocker, tmp_path):
```

Comment:
```suggestion
    def test_suggest_copy_working_dir(self, space: Space, mocker, tmp_path: Path):
```
It would also be nice to type the `mocker` if possible. (The type is `MockerFixture`, from `pytest_mock.plugin`)